include/sql_common.h

```diff
@@ -210,15 +210,7 @@ struct MYSQL_METHODS {
                         unsigned int field_count);
   void (*flush_use_result)(MYSQL *mysql, bool flush_all_results);
   int (*read_change_user_result)(MYSQL *mysql);
-<<<<<<< HEAD
 #if !defined(MYSQL_SERVER) || defined(XTRABACKUP)
```

Comment:
I noticed that here
https://github.com/percona/percona-xtrabackup/pull/1541/commits/b2c90eedf8d4d1c03b6dd37fa0bf7805b075840e#diff-2fd75cc82de9094dcf3f93b0c83caaff17b42e2b76ae7e9b23e9e679a648b444R47
the condition is combined
`#if (!defined(MYSQL_SERVER) && !defined(MYSQL_COMPONENT)) || defined(XTRABACKUP)`

Reply:
Done